plugins/plugin-docker/che-plugin-openshift-client/src/main/java/org/eclipse/che/plugin/openshift/client/OpenShiftConnector.java

```diff
@@ -1428,4 +1430,34 @@ private boolean isTerminalAgentInjected(final Set<String> exposedPorts) {
     private boolean isDevMachine(final Set<String> exposedPorts) {
         return exposedPorts.contains(CHE_WORKSPACE_AGENT_PORT + "/tcp");
     }
+
+    @Override
+    public SystemInfo getSystemInfo() throws IOException {
```

Comment:
I think we should move the method up, to come right after `startExec()`. That way, the code is easier to read and all public methods are grouped together.